src/Confluent.Kafka/IProducer.cs

```diff
@@ -391,13 +407,16 @@ public interface IProducer<TKey, TValue> : IClient
         /// <param name="timeout">
         ///     The maximum length of time this method may block.
         /// </param>
+        /// <exception cref="KafkaException">
+        ///     Thrown on all errors.
+        /// </exception>
```

Comment:
Is `KafkaTxnRequiresAbortException` possible here?

Reply:
Nope, see https://github.com/edenhill/librdkafka/blob/master/src/rdkafka.h#L6883 and make sure your docs match.